apps/admission-webhook-k8s/mutating-webhook-config.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook
```

Comment:
I think we should change the name to something like `nsm-mutating-webhook` or just `nsm-webhook`.

Webhooks are cluster-wide resources, so using name prefix (or suffix) is required to distinguish them.

Reply:
Thanks, done